talaiot/src/main/kotlin/com/cdsap/talaiot/publisher/InfluxDbPublisher.kt

```diff
@@ -1,10 +1,18 @@
 package com.cdsap.talaiot.publisher
 
 import com.cdsap.talaiot.configuration.InfluxDbPublisherConfiguration
-import com.cdsap.talaiot.entities.TaskMeasurementAggregated
+import com.cdsap.talaiot.entities.ExecutionReport
 import com.cdsap.talaiot.logger.LogTracker
 import com.cdsap.talaiot.request.Request
+import okhttp3.OkHttpClient
+import org.influxdb.InfluxDB
```

Comment:
one of the original ideas was don't include too many dependencies in the plugin, but with the GraphPublshers we broke the rule, so is ok

Reply:
I get this, but implementing fast networking with batching of metrics is not so easy. We can think of proguarding plugin maybe? :)

Reply:
I explained myself wrong. I mean is ok